### PR TITLE
Include more default hdfs libs

### PR DESCRIPTION
**Pull request by yuzhu**
by default, include 2.8, 2.9, 3.2 for emr and dataproc

### CODE REVIEW

File: dev/scripts/src/alluxio.org/build-distribution/cmd/common.go [hunk 1]
```diff
@@ -55,10 +55,11 @@ var ufsModules = map[string]module{
 	"ufs-hadoop-2.5": {"hadoop-2.5", false, "-pl underfs/hdfs -Pufs-hadoop-2 -Dufs.hadoop.version=2.5.2"},
 	"ufs-hadoop-2.6": {"hadoop-2.6", false, "-pl underfs/hdfs -Pufs-hadoop-2 -Dufs.hadoop.version=2.6.5 -PhdfsActiveSync"},
 	"ufs-hadoop-2.7": {"hadoop-2.7", true, "-pl underfs/hdfs -Pufs-hadoop-2 -Dufs.hadoop.version=2.7.3 -PhdfsActiveSync"},
-	"ufs-hadoop-2.8": {"hadoop-2.8", false, "-pl underfs/hdfs -Pufs-hadoop-2 -Dufs.hadoop.version=2.8.0 -PhdfsActiveSync"},
+	"ufs-hadoop-2.8": {"hadoop-2.8", true, "-pl underfs/hdfs -Pufs-hadoop-2 -Dufs.hadoop.version=2.8.0 -PhdfsActiveSync"},
+	"ufs-hadoop-2.9": {"hadoop-2.9", true, "-pl underfs/hdfs -Pufs-hadoop-2 -Dufs.hadoop.version=2.9.0 -PhdfsActiveSync"},
```
Comment (LuQQiu) [inline dev/scripts/src/alluxio.org/build-distribution/cmd/common.go:59]: Do we consider the minor version, for dataproc, it will be 2.9.2 and 2.8.5

Comment (yuzhu) [reply dev/scripts/src/alluxio.org/build-distribution/cmd/common.go]: done

**Comment by yuzhu** [comment]
@LuQQiu PTAL

**Comment by yuzhu** [comment]
alluxio-bot, merge this please

**Pull request merged**